File: src/Services/MealService/Domain/Meal.Domain/Entities/Meal.cs
using Meal.Domain.Common;
using Meal.Domain.Enums;

namespace Meal.Domain.Entities
{
    public class Meal : BaseEntity
    {
        public Meal() 
        {
            Foods = new List<Food>();
        }

        public MealType MealType { get; set; }
        public int TotalCalorie { get; set; }

        public ICollection<Food> Foods { get; set; }
    }
}

File: src/WebApps/Angular/AdminClient/src/app/contracts/meal/meal/meal.ts
import { Food } from "../food/food";

export class Meal {
    id: String;
    userId: String;
    status: number;
    createdDate: Date;
    updatedDate: Date | null | undefined;
    deletedDate: Date | null | undefined;
    mealType: number;
    totalCalorie: number;
    foods: Food[];
}